packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoTableHeaderCell/styles.ts

```diff
@@ -81,11 +81,7 @@ export const getWfoBasicTableStyles = ({ theme }: WfoTheme) => {
         display: 'flex',
         justifyContent: 'flex-start',
         alignItems: 'center',
-
-        [`.${HEADER_CELL_TITLE_BUTTON_CLASS}:has(+ .${HEADER_CELL_SORT_BUTTON_CLASS}:focus-visible)`]:
-            {
-                overflow: 'hidden',
-            },
+        overflow: 'hidden',
```

Comment:
This probably breaks the functionality where the sort button should be visible when colums are less wide than the header text uses

Reply:
Checked. Still seems to work